dss/operations/storage.py

```diff
@@ -0,0 +1,123 @@
+"""
+Storage consistency checks
+"""
+import json
+import logging
+from uuid import uuid4
+
+from cloud_blobstore import BlobNotFoundError
+
+from dss import Config, Replica
+from dss.storage.hcablobstore import compose_blob_key
+from dss.operations import dispatch
+from dss.operations.util import CommandForwarder, map_bucket
+from dss.events.handlers.sync import dependencies_exist
+
+
+logger = logging.getLogger(__name__)
+
+
+class StorageOperationHandler:
+    def __init__(self, argv, args):
```

Comment:
What is the difference between `argv` and `args` here?

Reply:
typing annotations added